SUMMARY.md

```diff
@@ -14,6 +14,10 @@
 - [Purchase A Bond (1, 1)](using-the-website/bonds.md)
 - [Stake LP Tokens](using-the-website/staking_lp.md)
 
+## FRAX Partnership
```

Comment:
I feel like this will not scale. Could we frame it like "utilise partner protocols" or something along these lines?

Reply:
Maybe we keep this first? When there are new partnerships announced we will see how to arrange the section better

Reply:
Let's get some feedback in Discord then. The Education team isn't much active on Github.

Reply:
I think it is more important to iterate faster rather than making it perfect, so that our content can keep up with the development pace

Reply:
I created [another PR](https://github.com/OlympusDAO-Education/Documentation/pull/52) which introduces a general Partnership section. Let me know what you think.